web-app/src/main/java/com/microsoft/azure/java/samples/moviedb/web/MovieController.java

```diff
@@ -35,6 +36,9 @@
     private static final Logger logger = LoggerFactory.getLogger(MovieController.class);
 
     @Autowired
+    private ApplicationContext applicationContext;
```

Comment:
Can we directly declare an auto-wired `CloudStorageAccount`?

Reply:
MovieController is by default singleton, which is not thread-safe. And CloudStorageAccount is not guaranteed to be thread-safe according to azure storage team.